test/unit/model/define.test.js

```diff
@@ -13,37 +13,84 @@ const dialect = Support.getTestDialect();
 describe(Support.getTestDialectTeaser('Model'), () => {
   describe('define', () => {
     it('should allow custom timestamps with underscored: true', () => {
-      const Model = current.define('User', {}, {
+      const User = current.define('User', {}, {
         createdAt: 'createdAt',
         updatedAt: 'updatedAt',
         timestamps: true,
         underscored: true,
       });
 
-      expect(Model.rawAttributes).to.haveOwnProperty('createdAt');
-      expect(Model.rawAttributes).to.haveOwnProperty('updatedAt');
+      expect(User.rawAttributes).to.haveOwnProperty('createdAt');
+      expect(User.rawAttributes).to.haveOwnProperty('updatedAt');
 
-      expect(Model._timestampAttributes.createdAt).to.equal('createdAt');
-      expect(Model._timestampAttributes.updatedAt).to.equal('updatedAt');
+      expect(User._timestampAttributes.createdAt).to.equal('createdAt');
+      expect(User._timestampAttributes.updatedAt).to.equal('updatedAt');
 
-      expect(Model.rawAttributes).not.to.have.property('created_at');
-      expect(Model.rawAttributes).not.to.have.property('updated_at');
+      expect(User.rawAttributes).not.to.have.property('created_at');
+      expect(User.rawAttributes).not.to.have.property('updated_at');
     });
 
-    it('should throw when id is added but not marked as PK', () => {
+    it('should throw only when id is added but primaryKey is not set', () => {
       expect(() => {
         current.define('foo', {
           id: DataTypes.INTEGER,
         });
-      }).to.throw('A column called \'id\' was added to the attributes of \'foos\' but not marked with \'primaryKey: true\'');
+      }).to.throw('An attribute called \'id\' was defined in model \'foos\' but primaryKey is not set. This is likely to be an error, which can be fixed by setting its \'primaryKey\' option to true. If this is intended, explicitly set its \'primaryKey\' option to false');
 
       expect(() => {
         current.define('bar', {
           id: {
             type: DataTypes.INTEGER,
+            primaryKey: true,
           },
         });
-      }).to.throw('A column called \'id\' was added to the attributes of \'bars\' but not marked with \'primaryKey: true\'');
+      }).to.not.throw('An attribute called \'id\' was defined in model \'bars\' but primaryKey is not set. This is likely to be an error, which can be fixed by setting its \'primaryKey\' option to true. If this is intended, explicitly set its \'primaryKey\' option to false');
```

Comment:
This is not a good way to check that the function will not throw because it could throw with a slightly different message and the test will pass.

Move this to its own test and simply check that the field 'id' has been added to rawAttributes and is a primary key.